### PR TITLE
Refine/cardtitle

### PR DESCRIPTION
**Pull request by derjubu**
Part of #8 
Create component for Cardtitle and implemented it in all instances

### CODE REVIEW

File: src/app/GlobalStyles.tsx [hunk 1]
```diff
@@ -5,6 +5,10 @@ const GlobalStyle = createGlobalStyle`
   box-sizing: border-box;
 }
 
+:root {
+  --background-title: #0000ffff;
+}
+
 body {
   margin: 10px;
     font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
```
Comment (davidwobido) [inline src/app/GlobalStyles.tsx:14]: Formatting looks strange, is stylelint working?

File: src/app/GlobalStyles.tsx [hunk 2]
```diff
@@ -5,6 +5,10 @@ const GlobalStyle = createGlobalStyle`
   box-sizing: border-box;
 }
 
+:root {
+  --background-title: #0000ffff;
```
Comment (davidwobido) [inline src/app/GlobalStyles.tsx]: Looks like here are to many 0s or fs

**Review by davidwobido — APPROVED** [review]
lgtm 💥 just 2 comments to your GlobalStyles.tsx

File: src/app/GlobalStyles.tsx (hunk 1 above)
Comment (lajanzen) [reply src/app/GlobalStyles.tsx:14]: Could also be prettier that's not working :)

File: src/app/components/CardTitle/CardTitle.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  justify-self: center;
+  grid-column: 1/-1;
+  width: 100%;
+  margin: 0;
+  font-size: 1.5rem;
+  color: white;
+  background-color: var(--background-title);
+  margin-bottom: 0.5rem;
+  padding-top: 0.5rem;
+  padding-bottom: 0.5rem;
```
Comment (lajanzen) [inline src/app/components/CardTitle/CardTitle.tsx:24]:
```suggestion
padding: 0.5rem 0;
```

**Review by lajanzen — COMMENTED** [review]
🚀

**Pull request merged**